text-ui-test/input.txt

```diff
@@ -5,6 +5,14 @@ add n/Daniela Koh d/met at SoC canteen e/other_email@u.nus.edu f/FoS
 list
 find Daniela
 find Koh
+find
```

Comment:
Good update of your test cases at every feature to better deal with regression testing.